charts/topolvm/templates/controller/deployment.yaml

```diff
@@ -194,6 +194,7 @@ spec:
             - --leader-election-namespace={{ .Release.Namespace }}
             {{ end }}
             - --http-endpoint=:9810
+            - --timeout=300s
```

Comment:
This is so that ControllerExpandVolume gets a 5min timeout. I initially did this for debugging and will remove it again, however I suspect this will lead to retries on the ControllerExpandVolume call. When reading through the code again, it becomes apparent that the first time we enter ControllerExpandVolume, we wait for LogicalVolume to be resized, after this we do not.